package.json

```diff
@@ -57,12 +57,14 @@
     },
     "plugins": [
       "@typescript-eslint",
-      "header"
+      "header",
+      "@fintechstudios/eslint-plugin-chai-as-promised"
```

Comment:
eslint is generally pretty smart. Does shortening this to `@fintechstudios/chai-as-promised` still work?